packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -106,7 +107,11 @@ export function CrossmintAuthProvider({ embeddedWallets, children, appearance }:
                 status: getAuthStatus(),
             }}
         >
-            <CrossmintWalletProvider defaultChain={embeddedWallets.defaultChain}>
+            <CrossmintWalletProvider
+                defaultChain={embeddedWallets.defaultChain}
+                showWalletModals={embeddedWallets.showWalletModals ?? true} // enabled by default
```

Comment:
if the default is set to `= true` [here](https://github.com/Crossmint/crossmint-sdk/pull/784/files#diff-c5100b9aa596950b4758a14a50a50f71c64f8b742801b6d3ea98e4f0a24a7febR60), then do we still need the `?? true`?

Reply:
Good call, i removed